### PR TITLE
feat: add descriptions to the UI

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Make the UI easier to understand.
- Detects aviutl that has been added manually.
- Displays information to help you decide which version to install.
- Clarifies how to add packages.

## How Has This Been Tested?

You can see it visually.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -363,7 +363,7 @@ <h2>最新バージョン</h2>
 							<textarea
 								class="form-control"
 								id="extra-data-url"
-								placeholder="追加データファイル取得先"
+								placeholder="追加プラグイン&スクリプト取得先（例: https://example.com/packages_list.xml）"
```
Comment (hal-shu-sato) [inline src/index.html:366]: I think “追加プラグイン&スクリプト取得先” looks complicated.
But I don't have a good idea.

File: src/index.html [hunk 2]
```diff
@@ -379,7 +379,7 @@ <h2>最新バージョン</h2>
 								class="btn btn-primary w-100"
 								id="set-extra-data-url"
 							>
-								追加データファイル取得先を設定
+								追加プラグイン&スクリプト取得先を設定
```
Comment (hal-shu-sato) [inline src/index.html:382]: Same as Line 366.

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
I felt some difficulty understanding.

File: src/index.html (hunk 1 above)
Comment (mitosagi) [reply src/index.html:366]: How about this?
データファイル取得先⇒データ取得先
追加データファイル取得先⇒追加パッケージ取得先

Comment (hal-shu-sato) [reply src/index.html:366]: Oh, looks better.
I think データ取得先 is good.

So, how about 追加データ取得先?
Maybe it's confusing...?

**Comment by mitosagi** [comment]
I fixed it.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for your improvement!!

**Pull request merged**